### PR TITLE
auth48 tiny editorial nits

### PR DESCRIPTION
**Pull request by kazuho**
I'm not sure if 1c10233 is applicable.

### CODE REVIEW

File: draft-ietf-httpbis-priority.md [hunk 1]
```diff
@@ -784,12 +783,13 @@ request ordering to influence response order.
 
 Incremental responses of the same urgency SHOULD be served by sharing bandwidth
 among them. The message content of incremental responses is used as parts, or
-chunks, are received. A client might benefit more from receiving a portion of
-all these resources rather than the entirety of a single resource. How large a
-portion of the resource is needed to be useful in improving performance varies.
-Some resource types place critical elements early; others can use information
-progressively. This scheme provides no explicit mandate about how a server
-should use size, type, or any other input to decide how to prioritize.
+chunks, as they are received. A client might benefit more from receiving a
```
Comment (LPardue) [inline draft-ietf-httpbis-priority.md:786]: I don't follow the proposed sentence

> The message content of incremental responses is used as parts, or chunks, as they are received.

The main issue is the double occurence of "as".

My intention was to say "used in parts, as they are received". But it seems you have a different intent here. Maybe something like "The message content of incremental responses can be used as soon as parts, or chunks, of the content is received"?

File: draft-ietf-httpbis-priority.md [hunk 2]
```diff
@@ -298,7 +298,7 @@ Servers can send SETTINGS_NO_RFC7540_PRIORITIES with a value of 1 to indicate
 that they will ignore HTTP/2 priority signals sent by clients.
 
 Endpoints that send SETTINGS_NO_RFC7540_PRIORITIES are encouraged to use
-alternative priority signals (for example, see {{header-field}} or
+alternative priority signals (for example, see Sections {{header-field}} or
```
Comment (LPardue) [inline draft-ietf-httpbis-priority.md:301]: the general RFC style seems to be:

Sections X and Y

Section X or Section Y

This make logical sense to me, so I would drop this change.

File: draft-ietf-httpbis-priority.md (hunk 1 above)
Comment (kazuho) [reply draft-ietf-httpbis-priority.md]: Thanks. I had problem parsing the new text, but rereading it, I think it works. Reverted the proposed change in 84c4b9a.

File: draft-ietf-httpbis-priority.md (hunk 2 above)
Comment (kazuho) [reply draft-ietf-httpbis-priority.md]: Thank you for the clarification. Reverted in 3ae5fd7.

**Pull request merged**